test/functionalTest/cases/2339_json_native_types_in_v1/json_native_types_in_v1.test

```diff
@@ -274,29 +274,29 @@ Date: REGEX(.*)
                     {
                         "metadatas": [
                             {
```

Comment:
Note that the changes in this .test are only on the ordering of metadata (note the Content-Lenght hasn't changed).